aewb_logger/aewb_logger_sender.c

```diff
@@ -3,8 +3,56 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <time.h>
+#include <errno.h>
+
+void get_ip_port_with_envvar_override(const char *in_dest_ip, in_port_t in_dest_port, bool *out_enable, const char **out_dest_ip, in_port_t *out_dest_port) {
+    const char *env_aewb_log_en_str = getenv("AEWB_LOG_EN");
+    const char *env_aewb_log_dest_ip_str = getenv("AEWB_LOG_DEST_IP");
+    const char *env_aewb_log_dest_port_str = getenv("AEWB_LOG_DEST_PORT");
+    long env_aewb_log_dest_port = 0;
+    
+    if (env_aewb_log_en_str && strcmp(env_aewb_log_en_str, "0")==0) {
```

Comment:
```suggestion
    if (env_aewb_log_en_str && strcmp(env_aewb_log_en_str, "0") == 0U) {
```

Reply:
`strcmp` retval is not unsigned (let's discuss all of these unsigned literal suggestions verbally).